core/jazz_environment-event-handler/index.js

```diff
@@ -591,6 +624,214 @@ function triggerBuildJob(payload, serviceDetails, configData) {
   });
 }
 
+function createSafe(environmentPayload, service_id, configData, authToken) {
+  return new Promise((resolve, reject) => {
+    if (!configData.TVAULT && !configData.TVAULT.IS_ENABLED) {
```

Comment:
Do this check in the generic method for the safe related activities rather than doing in all the methods

Reply:
provided check in common function addSafe.